packages/contentstack-bulk-publish/src/consumer/publish.js

```diff
@@ -25,13 +24,29 @@ function initializeLogger(fileName) {
 /* eslint-disable camelcase */
 function removePublishDetails(elements) {
   if (elements && elements.length > 0) {
-    return elements.map(({ _publish_details, ...rest }) => rest);
+    elements.map((element) => {
+      if (element.hasOwnProperty('publish_details')) {
+        delete element.publish_details;
+      }
+      return element;
+    });
```

Comment:
@cs-raj , [Minor]deleted line also work you just need to use publish_details instead of _publish_details.

Reply:
Used